draft-ietf-scitt-architecture.md

```diff
@@ -831,7 +878,7 @@ One of the following:
 - Status 404 - Entry not found.
   - Error code: `entryNotFound`
 
-The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document in the unprotected header, see draft-birkholz-scitt-receipts ([TODO]: replace with final reference).
+The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document in the unprotected header.
```

Comment:
```suggestion
The retrieved Receipt may be embedded in the corresponding COSE_Sign1 document in the unprotected header, of the signed statement.
```

Reply:
I am okay with that re-statement. Maybe add a backward reference to the Terminology Section later?